.maestro/custom_tabs/custom_tabs_navigation.yaml

```diff
@@ -0,0 +1,64 @@
+appId: com.duckduckgo.mobile.android
+name: "Custom Tabs navigation"
+tags:
+  - customTabsTest
+---
+- launchApp:
+    clearState: true
+    stopApp: true
+
+- assertVisible:
+    text: ".*Not to worry! Searching and browsing privately.*"
+- tapOn: "let's do it!"
+- runFlow:
+    when:
+      visible: "set as default"
+    commands:
+      - tapOn: "duckduckgo"
+      - tapOn: "set as default"
+      - assertVisible:
+          text: ".*I'll also upgrade the security of your connection if possible.*"
```

Comment:
we can replace this with `- runFlow: ../shared/onboarding.yaml`

Reply:
Since we need to make DDG the default browser, subsequent tests will have a different onboarding flow (The set default browser dialog won't be present).

So this only executes that step if the dialog is present.

Reply:
Makes sense!